controllers/clusterurlmonitor/clusterurlmonitor_supplement.go

```diff
@@ -90,7 +101,13 @@ func (s *ClusterUrlMonitorReconciler) EnsureServiceMonitorExists(clusterUrlMonit
 			return utilreconcile.RequeueReconcileWith(err)
 		}
 	}
-
+	condition, err := s.FindStatusCondition(hcp.Status.Conditions, string(metav1.ClusterVersionAvailable))
```

Comment:
`FindStatusCondition` returns `*metav1.Condition`, but here you're expecting `condition, err`